src/machine_helpers.py

```diff
@@ -28,3 +35,26 @@ def update_mongod_service(
     if role == Config.Role.CONFIG_SERVER:
         mongos_start_args = get_mongos_args(config, snap_install=True)
         add_args_to_env("MONGOS_ARGS", mongos_start_args)
+
+
+def setup_logrotate_and_cron() -> None:
+    """Create and write the logrotate config file."""
```

Comment:
In this doc string can you specify:
1. how often the cron job runs (I believe this is 1min)?
2. at what point do we rotate (I belive after 100MB?)

Reply:
done